mov_cli/media/quality.py

```diff
@@ -54,8 +54,12 @@ def get_quality(url: str | Path) -> Quality | None:
 
     if stream:
         height = stream[0]["height"]
+        width = stream[0]["width"]
 
         if height in Quality._value2member_map_:
             return Quality(height)
+        
+        if width in Quality._value2member_map_:
+            return Quality(width)
```

Comment:
this should work™

Reply:
how about if we get a resolution of 1080 x 480 💀

Reply:
it will get set to full HD when it's actually SD

![fdfd](https://media.tenor.com/TP12G5jWn24AAAAi/nerd.gif)

Reply:
> how about if we get a resolution of 1080 x 480 💀

not my issue.
![1214308056448630857](https://github.com/mov-cli/mov-cli/assets/132799819/35c02c93-395e-4978-9b71-13708a7ad0c5)

Reply:
but now that I think of it I have no idea how to deal with this but I suspect it's just a math equation.

Reply:
i ain't making math.

![image](https://github.com/mov-cli/mov-cli/assets/132799819/97818624-cdc2-4d0b-8901-9070e0e1966d)

Reply:
> how about if we get a resolution of 1080 x 480 💀

Also that would be an weird resolution to watch a video

Reply:
We could make it get the lowest number and get the resolution based on that.

Reply:
> We could make it get the lowest number and get the resolution based on that.

oh yeah true, I don't see that failing